cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bisonai/orakl-cli",
-  "version": "0.6.2",
+  "version": "0.6.3",
```

Comment:
hmm incrementing the last digit in versioning means we are patching a bug fix. I recommend updating to `1.0.0` since we are making backward-incompatible changes